code/tika/TikaRestClient.php

```diff
@@ -5,20 +5,58 @@
 
 class TikaRestClient extends Client
 {
+    /*
+    * @var Array
+    */
+    private static $_options = ['username' => null, 'password' => null];
```

Comment:
If these are going to be configuration options, we can drop the leading underscores and add `@config` to the doc blocks. 

The phpdoc should look like this:

```php
/**
 * Describe what it is
 * 
 * @config
 * @var array
 */
```

(note casing on Array vs array)

Same would apply to the others below